specification.markdown

```diff
@@ -223,6 +223,7 @@ Within an EBML Schema the `<element>` uses the following attributes to define an
 | attribute name | required | definition |
 |:---------------|:---------|:-----------|
 | name           | Yes      | The official human-readable name of the EBML Element. The value of the name MUST be in the form of an NCName as defined by the [XML Schema specification](http://www.w3.org/TR/1999/REC-xml-names-19990114/#ns-decl). |
+| parent         | Yes      | The name of the parent EBML Elements. This value MUST be defined with the full hierarchy of EBML Elements separated with a dot. The top EBML Element in the hierarchy being the first in the value. Elements at the top level MUST use the 'none' parent value. Elements that can be found anywhere in the EBML Body or Header MUST use the 'any' parent value. |
```

Comment:
For the root element (currently known as level 0) it would be logical to have the parent as `/`. An empty value for elements that can be found anywhere (like Void and CRC32) would make sense too.